phpBB/phpbb/avatar/driver/local.php

```diff
@@ -182,7 +184,7 @@ protected function get_avatar_list($user)
 					}
 					$cat = ($path == $file_path) ? $user->lang['NO_AVATAR_CATEGORY'] : str_replace("$path/", '', $file_path);
 					$avatar_list[$cat][$image] = array(
-						'file'      => ($cat != $user->lang['NO_AVATAR_CATEGORY']) ? rawurlencode($cat) . '/' . rawurlencode($image) : rawurlencode($image),
+						'file'      => ($cat != $user->lang['NO_AVATAR_CATEGORY']) ? str_replace('%2F', '/', rawurlencode($cat)) . '/' . rawurlencode($image) : rawurlencode($image),
```

Comment:
No. The image list is retrieved from the filesystem. The user can only selected those files. Trying to select categories that do not exist in this list or images that do not exist in that list, e..g by using ```../../``` will result in an empty list for the categories and an error message for the image.